test/integration/scheduler/framework_test.go

```diff
@@ -448,7 +448,7 @@ func TestPreFilterPlugin(t *testing.T) {
 	// Create the master and the scheduler with the test plugin set.
 	context := initTestSchedulerWithOptions(t,
 		initTestMaster(t, "prefilter-plugin", nil),
-		false, nil, registry, plugins, emptyPluginConfig, false, time.Second)
+		false, nil, registry, plugins, emptyPluginConfig, false)
```

Comment:
is it a good idea if you specify the sync period as `0` explicitly? rather than assuming the default value.

Reply:
Setting to `0` is a common pattern and I believe it's the default value. And again, the "sync period" is NOT syncing with apiserver. See:

- #75622
- https://github.com/kubernetes/kubernetes/issues/75495#issuecomment-475629555

Reply:
And setting to 0 doesn’t mean never resync, my memory is that it underneath sets it to a random 10ish minute.